internal/venice-common/src/main/java/com/linkedin/venice/kafka/consumer/AutoClosingKafkaConsumer.java

```diff
@@ -322,6 +322,30 @@ public OffsetAndMetadata committed(TopicPartition partition, Duration timeout) {
     }
   }
 
+  @Override
+  public Map<TopicPartition, OffsetAndMetadata> committed(Set<TopicPartition> partitions) {
+    if (partitions == null || partitions.isEmpty()) {
```

Comment:
Are these methods being used by Venice?

Reply:
Actually, we don't use the AutoClosingKafkaConsumer at all. It was a tactical fix that we ended up not using because a cleaner fix was implemented inside the Kafka client itself... I plan to delete it, but if you want to do it as part of this PR that would be great too.